app/javascript/dashboard/components/widgets/conversation/AssigneeTypeFilter.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <div>
+    <woot-button
+      variant="smooth"
+      size="small"
+      color-scheme="secondary"
+      class="selector-button"
+      @click="toggleDropdown"
+    >
+      {{ $t('CHAT_LIST.CHAT_ASSIGNEE_TYPE_FILTER_ITEMS')[activeType]['TEXT'] }}
+      <span class="assignee-type--count count">{{ activeTypeCount }}</span>
+      <fluent-icon
+        :icon="showActionsDropdown ? 'chevron-up' : 'chevron-down'"
+        class="icon"
+        size="16"
+      />
+    </woot-button>
+    <div
+      v-if="showActionsDropdown"
+      v-on-clickaway="closeDropdown"
+      class="dropdown-pane dropdown-pane--open"
+    >
+      <woot-dropdown-menu>
+        <woot-dropdown-sub-menu
+          :title="this.$t('CHAT_LIST.CHAT_ASSIGNEE_TYPE_FILTER')"
```

Comment:
```suggestion
          :title="$t('CHAT_LIST.CHAT_ASSIGNEE_TYPE_FILTER')"
```

Don't need the `this`